arm-service-map/2015-11-01-preview/swagger/arm-service-map.json

```diff
@@ -0,0 +1,2508 @@
+          "type": "object",
+          "description": "Resource properties.",
+          "properties": {
+            "timestamp": {
+              "type": "string",
+              "format": "date-time",
+              "description": "UTC date and time when this resource was updated in the system.         "
+            },
+            "monitoringState": {
+              "$ref": "#/definitions/MonitoringState"
+            },
+            "virtualizationState": {
+              "$ref": "#/definitions/VirtualizationState"
+            },
+            "displayName": {
+              "type": "string",
+              "description": "Name to use for display purposes"
+            },
+            "computerName": {
+              "type": "string",
+              "description": "Name of the machine, e.g., server"
+            },
+            "fullyQualifiedDomainName": {
+              "type": "string",
+              "description": "Fully-qualified name of the machine, e.g., server.company.com"
+            },
+            "bootTime": {
+              "type": "string",
+              "format": "date-time",
+              "description": "UTC date and time when the machine last booted"
+            },
+            "timezone": {
+              "$ref": "#/definitions/Timezone"
+            },
+            "agent": {
+              "$ref": "#/definitions/AgentConfiguration"
+            },
+            "resources": {
+              "$ref": "#/definitions/MachineResourcesConfiguration"
+            },
+            "networking": {
+              "$ref": "#/definitions/NetworkConfiguration"
+            },
+            "operatingSystem": {
+              "$ref": "#/definitions/OperatingSystemConfiguration"
+            },
+            "virtualMachine": {
+              "$ref": "#/definitions/VirtualMachineConfiguration"
+            },
+            "hypervisor": {
+              "$ref": "#/definitions/HypervisorConfiguration"
+            }
+          }
+        }
+      }
+    },
+    "Process": {
+      "description": "A process resource represents a process running on a machine. The process may be actively *monitored*, i.e., a Dependency Agent is running on its machine, or *discovered*, i.e., its existence was inferred by observing the data stream from monitored machines. A process resource represents a pool of actual operating system resources that share command lines and metadata. As the process pool evolves over time, prior versions of the process resource are preserved and available for access. A process is live during an interval of time, if that process is executing during (parts) of that interval",
+      "allOf": [
+        {
+          "$ref": "#/definitions/CoreResource"
+        }
+      ],
+      "x-ms-discriminator-value": "process",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "type": "object",
+          "description": "Resource properties.",
+          "properties": {
```

Comment:
same as above: General comment: Please describe the required and readOnly properties of a model accurately.